### PR TITLE
[forge-viewer] Update Property type

### PR DESCRIPTION
**Pull request by adrianmfi**
Properties can have numeric displayValue and nullable units: 
```
{
  "displayName": "Required",
  "displayValue": 0,
  "displayCategory": "Item",
  "attributeName": "Required",
  "type": 1,
  "units": null,
  "hidden": false,
  "precision": 0
}
```
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://forge.autodesk.com/en/docs/viewer/v7/reference/Viewing/Model/#getProperties/

### CODE REVIEW

File: types/forge-viewer/index.d.ts
Comment (adrianmfi) [reply types/forge-viewer/index.d.ts:859]: @liskaj Sure, thanks, added now.

**Review by liskaj — APPROVED** [review]

**Review by MysteryBlokHed — COMMENTED** [review]
If you add some tests for these changes (in `forge-viewer-tests.ts`), the PR can get merged without a DefinitelyTyped maintainer's approval 😄

**Review by liskaj — CHANGES_REQUESTED** [review]
@adrianmfi Can you fix according to comments above, pls?

**Comment by adrianmfi** [comment]
> @adrianmfi Can you fix according to comments above, pls?

@liskaj I've added an optional field precision and type assertions of Property in `forge-viewer-tests.ts`, I think that fixes the comments?

**Review by liskaj — APPROVED** [review]

**Comment by liskaj** [comment]
Ready to merge

**Pull request merged**
